pkg/daemon/controller/nodeinfo.go

```diff
@@ -384,15 +413,15 @@ func (r *nodeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	if feature.MultiClusterEnabled() {
 		if err := nodeController.Watch(&source.Kind{Type: &multiclusterv1.RemoteVtep{}},
-			&fixedKeyHandler{key: ActionReconcileNode},
+			&fixedKeyHandler{key: ActionReconcileNodeInfo},
 			predicate.Funcs{
 				UpdateFunc: func(updateEvent event.UpdateEvent) bool {
 					oldRemoteVtep := updateEvent.ObjectOld.(*multiclusterv1.RemoteVtep)
 					newRemoteVtep := updateEvent.ObjectNew.(*multiclusterv1.RemoteVtep)
 
 					if oldRemoteVtep.Spec.VTEPInfo.IP != newRemoteVtep.Spec.VTEPInfo.IP ||
```

Comment:
ditto, nil pointer check

Reply:
The VTEPInfo field of RemoteVtep is not a pointer.